### PR TITLE
Merge DNSSD into main Poco

### PR DESCRIPTION
**Pull request by soroshsabz**
ITNOA

Merge DNSSD into main Poco project

fixes https://github.com/pocoproject/poco/issues/2208

### CODE REVIEW

**Comment by soroshsabz** [comment]
@aleks-f @matejk as you say, I create new PR, so please start CI

thanks

**Review by aleks-f — CHANGES_REQUESTED** [review]
we need only `vs_90`, `vs_160` and `vs_170` projects here (see #4274). all others should be deleted, and `vs_160` and `vs_170` generated with [progen](https://github.com/pocoproject/poco/wiki/Generating-VS-Projects-using-ProGen)

**Comment by geirhei** [comment]
Any progress on this @soroshsabz? The library is very useful, would be great to get it into POCO.

**Comment by soroshsabz** [comment]
@aleks-f I am done

please run CI

thanks

**Comment by soroshsabz** [comment]
@matejk

**Comment by matejk** [comment]
I don't know why only a few of the CI jobs were executed and not all of them.

File: DNSSD/Bonjour/Bonjour_vs100.vcxproj [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (aleks-f) [inline DNSSD/Bonjour/Bonjour_vs100.vcxproj:1]: bonjour old vs projects should be removed and vs160/170 created

File: DNSSD/samples/DNSSDBrowser/DNSSDBrowser_WEC2013_vs120.vcxproj.filters [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (aleks-f) [inline DNSSD/samples/DNSSDBrowser/DNSSDBrowser_WEC2013_vs120.vcxproj.filters:1]: samples old vs projects should be removed and vs160/170 created

File: ActiveRecord/ActiveRecord_vs170.vcxproj [hunk 3]
```diff
@@ -230,7 +230,7 @@
   </ImportGroup>
   <PropertyGroup Label="UserMacros"/>
   <PropertyGroup>
-    <_ProjectFileVersion>17.0.34322.80</_ProjectFileVersion>
+    <_ProjectFileVersion>17.0.34511.75</_ProjectFileVersion>
```
Comment (aleks-f) [inline ActiveRecord/ActiveRecord_vs170.vcxproj:233]: seems like we're going backwards here? there is no reason to modify other vs projects in this PR

File: DNSSD/DNSSD_vs90.vcproj [hunk 4]
```diff
@@ -0,0 +1,436 @@
+<?xml version="1.0" encoding="Windows-1252"?>
```
Comment (aleks-f) [inline DNSSD/DNSSD_vs90.vcproj:1]: only *_vs90.vcproj is needed, other vs90 files should be removed

**Comment by soroshsabz** [comment]
@aleks-f I resolve all your notes

please run CI

**Comment by aleks-f** [comment]
#4479

**Comment by soroshsabz** [comment]
@aleks-f 

I am happy to merge that PR

thanks

**Pull request closed without merging**